entity-service-attribute-translator/build.gradle.kts

```diff
@@ -12,8 +12,8 @@ dependencies {
   implementation("com.typesafe:config:1.4.2")
   implementation("org.slf4j:slf4j-api:1.7.30")
```

Comment:
I updated the impl to slf4j2 but I don't think there is any module for api

Reply:
the "2" refers to the major version of the slf4j lib, so no module name change just a version one. That's actually why we want to update them all here, because it's easy to inherit a major version bump from a dependency.

```suggestion
  implementation("org.slf4j:slf4j-api:2.0.9")
```